src/styles/globals.css

```diff
@@ -108,8 +108,13 @@ body {
   border-top-color: #00857F !important;
 }
 
+
 .txlive-langselector {
-  @apply !top-0 !right-28 !bottom-auto !left-auto !bg-brand-800 !rounded-b-3xl !px-5 !uppercase !font-semibold !font-sans !print;
```

Comment:
This instruction was wrong, did you try !print: hidden? maybe that prevents as from using custom classes